compact_activity_snapshot.proto

```diff
@@ -48,12 +48,12 @@ message Backend {
     WAIT_EVENT_LWLOCK_WAL_BUF_MAPPING_LOCK = 107; // WALBufMappingLock
     WAIT_EVENT_LWLOCK_WAL_WRITE_LOCK = 108; // WALWriteLock
     WAIT_EVENT_LWLOCK_CONTROL_FILE_LOCK = 109; // ControlFileLock
-    WAIT_EVENT_LWLOCK_CHECKPOINT_LOCK = 110; // CheckpointLock
-    WAIT_EVENT_LWLOCK_C_LOG_CONTROL_LOCK = 111; // CLogControlLock
-    WAIT_EVENT_LWLOCK_SUBTRANS_CONTROL_LOCK = 112; // SubtransControlLock
+    WAIT_EVENT_LWLOCK_CHECKPOINT_LOCK = 110; // CheckpointLock (removed in 14)
+    WAIT_EVENT_LWLOCK_C_LOG_CONTROL_LOCK = 111; // CLogControlLock, XactSLRULock
```

Comment:
@lfittl I forgot if we decided to update the name here too like the following or not. Do you... remember?

```suggestion
    WAIT_EVENT_LWLOCK_XACT_SLRU_LOCK = 111; // CLogControlLock, XactSLRULock
```

Reply:
I think we said we'd update the enum key name it to what its now named, i.e. in the most recent Postgres version.

Reply:
cool cool 👍  I'll make a new commit to update all the things 🎉